CHANGELOG.md

```diff
@@ -4,7 +4,53 @@
 
 #### Changed
 
-- Internal CPU mining now injects nonce bytes at the end.
+## 1.7 (2019-03-26)
```

Comment:
If this is the new official entry, please reconcile the `## Unreleased` heading above this.

Reply:
Did you mean just remove the `## Unreleased` entry for now, while it's empty?

Reply:
Yes, since you moved the note about CPU mining.